### PR TITLE
Updated TOC to add inheritance topic.

### PR DESCRIPTION
**Pull request by rpetrusha**
# Update TOC to add inheritance topic

## Summary

Adds the new C# inheritance tutorial to the TOC.

## Suggested Reviewers

@mairaw @billwagner @stevehoag

### CODE REVIEW

File: docs/toc.md [hunk 1]
```diff
@@ -229,6 +229,7 @@
 ### [🔧 Asynchronous UI programming](csharp/tutorials/asynchronous-ui-programming.md)
 ### [🔧 Asynchronous server programming](csharp/tutorials/asynchronous-server-programming.md)
 ### [🔧 Concurrent programming](csharp/tutorials/concurrent-programming.md)
+### [inheritance in C# and .NET](csharp/tutorials/inheritance.md)
```
Comment (stevehoag) [inline docs/toc.md]: Shouldn't inheritance be capitalized?

Comment (mairaw) [inline docs/toc.md]: Nit: capital I

**Comment by rpetrusha** [comment]
@mairaw Thanks for noticing; I completely missed it. I've fixed it.

**Comment by mairaw** [comment]
LGTM

**Pull request merged**